src/init/init.js

```diff
@@ -40,58 +37,36 @@ async function initializeManifest() {
     log.info(`Loading: ${manifest.name} v${manifest.version}`);
 }
 
-function configI18n(keyUiLocale: string) {
-    const locale = keyUiLocale || 'en';
-    config.i18n.sources.add(`i18n/module/i18n_module_${locale}.properties`);
-    return keyUiLocale;
+function isLangRTL(code) {
+    const langs = ['ar', 'fa', 'ur']
+    const prefixed = langs.map(c => `${c}-`)
+    return langs.includes(code) || prefixed.filter(c => code.startsWith(c)).length > 0
 }
 
-// TODO: Make api-requests?
-function getLocaleSpecs(locale: string) {
```

Comment:
Right now we are just including one language from `locales` (en), but I think when we add additional ones we will get e.g. norwegian. I'm going to test this theory though.

```
// in app entry point
import './locales'

// to change locales
function changeLocale(locale) {
      moment.locale(locale)
      i18n.changeLanguage(locale)
      document.body.setAttribute('dir', isLangRTL(locale) ? 'rtl' : 'ltr')
}
```

Reply:
If it doesn't work it should be fixed in d2-i18n I think since it claims to:

>On dev/build phase src/locales directory would be created. It will contain necessary setup for internationalization, date/time, calendar etc. It is auto-generated, so please don't update it, any changes to it will be lost.

Reply:
Thanks for clarifying. I like your changes. So, as far as I understand, to make other locales work we need: - The pot file for the locale -make sure the locale file for dateFn exists